### DISCUSSION TITLE
Flow-compatible LED deck

### DISCUSSION THREAD

**Discussion by DavidHaitch** [bitcraze/discussions#1097] [Ideas]
I was just trying to do some light painting with the LED deck, and I was having trouble flying a nice smooth line. 
It occurred to me that a Flow deck would help a lot, but of course you can't mount both decks to the bottom of the drone. 
So what if the LED deck had a cutout in the middle for the Flow deck to look through? 
I saw some prior art suggesting that it was possible (https://www.hackster.io/krichardsson/light-paint-with-a-drone-d050af)
so maybe the LED ring should come with those holes by default.

**Comment by krichardsson** [comment]
Thanks for the suggestion! 
It would be possible, but requires a bit of work, usually more than expected :-)
We produce products in batches and we will consider it for future batches, I'm not sure when this might be.

**Reply by DavidHaitch** [reply]
Oh, I'm sure. It always does. For one thing, you might also need to start selling longer landing legs!
